public/app/features/playlist/partials/playlist.html

```diff
@@ -27,7 +27,7 @@ <h3 class="page-sub-heading" ng-show="ctrl.isNew">New Playlist</h3>
   </div>
 
   <div class="gf-form-group">
-    <h3 class="page-headering">Dashboards</h3>
+    <h3 class="page-heading">Dashboards</h3>
```

Comment:
This was a cross between `page-header` and `page-heading` the two are very similar -- it took me a long time to decide which was correct -- and initially I picked the wrong one. I think.